addons/csi/digitalocean/csi-driver.yaml

```diff
@@ -118,7 +118,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-do-plugin
-          image: '{{ Registry "docker.io" }}/digitalocean/do-csi-plugin:v4.2.0'
+          image: '{{ Image "digitalocean/do-csi-plugin:v4.2.0" }}'
```

Comment:
Why don't we keep `docker.io` here?

Reply:
Because the function will prepend it by default anyway. To me the practice of prepending it manually was always a KKP-ism, necessary for easy search&replace registry changes. But with the new code, it's not necessary anymore, so I generally removed the docker.io/ prefix.